### DISCUSSION TITLE
how to draw circle using bevy ui?

### DISCUSSION THREAD

**Discussion by Nigless** [bevyengine/bevy#13505] [General]
Im making 3D FPS game and i need to show crosshair on screen.

**Comment by doonv** [comment]
You could use a [`UiMaterial`](https://docs.rs/bevy/0.13.2/bevy/prelude/trait.UiMaterial.html) and use a shader to create draw a circle.

Or you could just slap an [`UiImage`](https://docs.rs/bevy/0.13.2/bevy/prelude/struct.UiImage.html) with an image of a circle on the screen, whatevers easier.
